server/src/main/java/org/elasticsearch/search/runtime/LongScriptFieldRangeQuery.java

```diff
@@ -49,7 +63,7 @@ public final String toString(String field) {
             b.append(fieldName()).append(':');
         }
         b.append('[').append(lowerValue).append(" TO ").append(upperValue).append(']');
-        return b.toString();
+        return b.toString() + " approximated by " + approximation();  // TODO move the toString enhancement into the superclass
```

Comment:
I think `AbstractScriptFieldQuery` should handle this bit of the `toString` but that'd require touching every subclass and felt like it should be saved for a follow up.